src/sentry/api/endpoints/group_tagkey_values.py

```diff
@@ -29,6 +29,11 @@ def get(self, request: Request, group, key) -> Response:
         :pparam string key: the tag key to look the values up for.
         :auth: required
         """
+        analytics.record(
+            "eventuser_endpoint.request",
+            project_id=group.project_id,
+            endpoint="sentry.api.endpoints.group_tagkey_values.get",
```

Comment:
sorry if I'm just not seeing it, but where is `EventUser` used in this endpoint?